include/flatbuffers/grpc.h

```diff
@@ -287,8 +288,14 @@ template<class T> class SerializationTraits<flatbuffers::grpc::Message<T>> {
   }
 
   // Deserialize by pulling the
-  static grpc::Status Deserialize(grpc_byte_buffer *buffer,
+  static grpc::Status Deserialize(::grpc::ByteBuffer *bbuffer,
                                   flatbuffers::grpc::Message<T> *msg) {
+    
+    std::vector<Slice>* slices = new std::vector<Slice>;
+    bbuffer->Dump(slices);
+    grpc_byte_buffer* buffer = g_core_codegen_interface->grpc_raw_byte_buffer_create(
```

Comment:
This seems to involve extra copying and maybe allocation, we'd like to avoid that. Can this be done differently?